### PR TITLE
Type safety to fixtures

### PR DESCRIPTION
**Pull request by liywjl**
## What does this change?
Move fixtures into `src/`

## Why?
Add type safety to fixtures

### CODE REVIEW

File: src/lib/mockFetchCalls.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import fetchMock from "fetch-mock";
 
-import { discussion } from "../../fixtures/discussion";
-import { comment } from "../../fixtures/comment";
+import { discussion } from "../fixtures/discussion.ts";
```
Comment (oliverlloyd) [inline src/lib/mockFetchCalls.js]: Is the .ts needed?

File: src/fixtures/comment.ts [hunk 2]
```diff
@@ -50,7 +41,7 @@ export const comment = {
           "https://discussion.guardianapis.com/discussion-api/comment/138809272",
         isoDateTime: "2020-03-09T19:09:28Z",
         date: "09 March 2020 7:09pm",
-        commentId: "138809272",
+        commentId: 138809272,
```
Comment (oliverlloyd) [inline src/fixtures/comment.ts]: Is this part of the move or the start of something else?

File: src/types.tsx [hunk 3]
```diff
@@ -119,7 +119,7 @@ export interface UserProfile {
 
 export interface DiscussionResponse {
   status: string;
-  page: number;
+  currentPage: number;
```
Comment (oliverlloyd) [inline src/types.tsx]: Is this related to moving the file?

Comment (liywjl) [reply src/types.tsx]: I assume the fixtures come from actual API calls. If so there does not exist a `page` key only `currentPage`, so i made the relavant changes

File: src/fixtures/comment.ts (hunk 2 above)
Comment (liywjl) [reply src/fixtures/comment.ts]: for the sake of keeping this PR small im moving everything to number, in the next PR I will move them to all strings (right not they error types)

File: src/lib/mockFetchCalls.js (hunk 1 above)
Comment (liywjl) [reply src/lib/mockFetchCalls.js]: good catch
